docs/integrations/sources/recharge-migrations.md

```diff
@@ -0,0 +1,4 @@
+# Recharge Migration Guide
+
+## Upgrading to 2.0.0
+This version introduces no breaking changes, but the code-base changed from CDK Python to CDK Low-code. No actions are needed.
```

Comment:
So breaking change is better for users (they don't have to freak out because of the opt-in message) and for us (as versions are more aligned since all users are upgraded at the same time). I would avoid breaking change if possible